experimental/traffic-portal/src/app/core/certs/cert-viewer/cert-viewer.component.scss

```diff
@@ -0,0 +1,33 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+.tab-content {
+	margin-top: 5px;
+	display: grid;
+
+	&:is(form), .mat-accordion {
+		width: 98%;
+		margin: auto auto 15px;
+	}
+}
+
+.private-text {
+	color: transparent !important;
```

Comment:
> `!important`

Reply:
Unfortunately, this required regardless of where this css is used (`styles.scss`, `theme.scss`, etc) as Material assumes this is set only by the primary/accent color specified in the current theme.